include/rdmnet/common.h

```diff
@@ -354,28 +354,14 @@ typedef enum
   kRdmnetCCSetCommand = 0x30
 } rdmnet_command_class_t;
 
-/**
- * A set of identifying information for a network interface, for multicast purposes. RDMnet uses
- * two multicast protocols, LLRP and mDNS. When creating sockets to use with these protocols, the
- * interface IP addresses don't matter and the primary key for a network interface is simply a
- * combination of the interface index and the IP protocol used.
- */
-typedef struct RdmnetMcastNetintId
-{
-  /** The IP protocol used on the network interface. */
-  etcpal_iptype_t ip_type;
-  /** The index of the network interface. See @ref interface_indexes for more information. */
-  unsigned int index;
-} RdmnetMcastNetintId;
-
 /**
  * Network interface configuration information to give the RDMnet library at initialization. LLRP
  * multicast and discovery traffic will be restricted to the network interfaces given.
```

Comment:
Some of the deleted doc comment for RdmnetMcastNetintId contains some useful information that I think should be moved to this one. Namely:

> the interface IP addresses don't matter and the primary key for a network interface is simply a combination of the interface index and the IP protocol used.

Some rephrasing is probably needed to make it fit in to this comment.

Reply:
Just created https://github.com/ETCLabs/EtcPal/pull/43 for this.